src/components/computing-status/use-computing-status.ts

```diff
@@ -60,7 +70,7 @@ function isWorthUpdate(
     if (rootNetworkUuidFromNotification && rootNetworkUuidFromNotification !== currentRootNetworkUuid) {
         return false;
     }
-    if (fetcher && lastUpdateRef.current?.fetcher !== fetcher) {
+    if (computingStatusFetcher && lastUpdateRef.current?.computingStatusFetcher !== computingStatusFetcher) {
```

Comment:
Do we need to do the same thing for `computingStatusInfosFetcher` ?

Reply:
since they are linked, I'm assuming they will both change together
I'm not sure adding a case would improve the code